File: demo-vk-rest-scraping.java/src/main/java/demo/http/RandomUserAgent.java
package demo.http;

import com.google.common.collect.ImmutableList;
import demo.file.TextFile;

import java.util.Random;

class RandomUserAgent {

    private static final ImmutableList<String> USER_AGENTS;
    private static final Random RANDOM = new Random();

    static {
        USER_AGENTS = ImmutableList.copyOf(new TextFile("http/UserAgents.txt").loadAsIterable());
    }

    static String getRandomUserAgent() {
        return USER_AGENTS.get(RANDOM.nextInt(USER_AGENTS.size()));
    }
}

File: demo-vk-rest-scraping.kotlin/src/main/kotlin/demo/http/RandomUserAgent.kt
package demo.http

import com.google.common.collect.ImmutableList
import demo.file.TextFile
import java.util.*

internal object RandomUserAgent {

    private val USER_AGENTS: ImmutableList<String>
    private val RANDOM = Random()

    init {
        USER_AGENTS = ImmutableList.copyOf(TextFile("http/UserAgents.txt").loadAsIterable())
    }

    fun getRandomUserAgent(): String = USER_AGENTS[RANDOM.nextInt(USER_AGENTS.size)]
}
